providers/implementations/macs/kmac_prov.c

```diff
@@ -535,6 +535,9 @@ static int bytepad(unsigned char *out, size_t *out_len,
                    const unsigned char *in1, size_t in1_len,
                    const unsigned char *in2, size_t in2_len, size_t w)
 {
+    if (w == 0)
+        return 0;
+
```

Comment:
Declarations must be first in the function.

Reply:
The `if` statement is above the declarations that are on line 541. It needs to be below them.
Basically, C89 (which I still believe is the standard OpenSSL follows) does not allow mixing of declarations within code; all declarations must be at the start of a block. This change violates that.